### PR TITLE
Adds Brazilian Portuguese translation

### PR DESCRIPTION
**Pull request by bltavares**
This PR follows the TRANSLATIONS.md file to include the Brazilian Portuguese translation.

The translation is based on the `pt-pt.json` translation, with some minor adjustments to be closer to usually idioms used in Brasil.

This is required to have the Portuguese translation on pt-BR Android apps, otherwise it will fallback to English, as it does not consider the `pt-PT` translation.

### CODE REVIEW

File: src/locals/pt-br.json [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    "auto_thought_ex": "Me esqueci da festa do Jorge! Ele deve me odiar.",
+    "challenge_ex": "O Jorge sabe que não consigo ir a todas as festas. Eu não o odiaria caso se esquecesse da minha. Tenho que me considerar tão bem quanto considero os outros.",
+    "alt_thought_ex": "É verdade que me esqueci da festa do Jorge mas posso pedir desculpa e ele compreenderá.",
+    "cog_distortion": { "label": "Leitura da mente", "slug": "leitura-mente" },
+    "intro": {
+      "line1": "Bem vindo ao",
+      "emphasis1": "quirk",
+      "line2": ", uma",
+      "emphasis2": "app",
+      "line3": " de terapia cognitivo comportamental (TCC)."
```
Comment (wkurosawa) [inline src/locals/pt-br.json]: What if we Title Case "Terapia Cognitivo Comportamental"?
```suggestion
      "line3": " de Terapia Cognitivo Comportamental (TCC)."
```

File: src/locals/pt-br.json [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+      "line1": "Procure por",
+      "emphasis1": "distorções cognitivas"
+    },
+    "bad_distortions": {
+      "line1": "Sabemos mesmo se o Jorge vai nos odiar? Se não, estamos fazendo",
+      "emphasis1": "leitura da mente.",
+      "line2": "Me esqueci da festa do Jorge. Ele deve me odiar."
+    },
+    "challenge_screen": {
+      "line1": "3. Porque é que este pensamento não é realista?",
```
Comment (wkurosawa) [inline src/locals/pt-br.json]: Would this be better? 
```suggestion
      "line1": "3. Por que este pensamento não é realista?",
```

File: src/locals/pt-br.json [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+    "edit": "Editar",
+    "new": "Novo",
+    "save": "Salvar",
+    "auto_thought_placeholder": "O que está acontencendo?",
+    "cog_distortion": "Distorção cognitiva",
+    "challenged": "Desafie-o",
+    "changed_placeholder": "Debate o pensamento!",
+    "alt_thought_placeholder": "O que devemos pensar alternativamente?"
+  },
+  "all_or_nothing_thinking": "Pensamento dicotómico",
```
Comment (wkurosawa) [inline src/locals/pt-br.json]: Should the distortions Title Case? 

```suggestion
  "all_or_nothing_thinking": "Pensamento Dicotômico",
```

File: src/locals/pt-br.json [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+  "all_or_nothing_thinking_explanation": "Consiste na valorização de um acontecimento sem ter em conta possíveis aspectos ou resultados intermediários. Se pensarmos que uma pequena falha nossa significa que estamos fundamentalmente arruinados, então estamos provavelmente praticando pensamento dicotómico.",
```
Comment (wkurosawa) [inline src/locals/pt-br.json]:
```suggestion
  "all_or_nothing_thinking_explanation": "Consiste na valorização de um acontecimento sem ter em conta possíveis aspectos ou resultados intermediários. Se pensarmos que uma pequena falha nossa significa que estamos fundamentalmente arruinados, então estamos provavelmente praticando pensamento dicotômico.",
```

File: src/locals/pt-br.json [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+  "all_or_nothing_thinking_thought": "Falhei na entrevista, certamente não sou empregável",
+  "catastrophizing": "Catastrofização",
+  "catastrophizing_explanation": "Se estamos considerarando um pequeno problema e a aumentando drasticamente o seu significado, então estamos a catastrofizar. Se você cometeu um pequeno erro no trabalho e teme que alguém o descubra, mesmo não sendo nada de grave? Estás provavelmente catastrofizando.",
```
Comment (wkurosawa) [inline src/locals/pt-br.json]:
```suggestion
  "catastrophizing_explanation": "Se estamos considerarando um pequeno problema e a aumentando drasticamente o seu significado, então estamos a catastrofizar. Se você cometeu um pequeno erro no trabalho e teme que alguém o descubra, mesmo não sendo nada de grave? Provavelmente você está catastrofizando.",
```

File: src/locals/pt-br.json [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+  "catastrophizing_thought": "Estou agitado, devo estar tendo um ataque cardíaco",
+  "emotional_reasoning": "Raciocínio emocional",
```
Comment (wkurosawa) [inline src/locals/pt-br.json]:
```suggestion
  "emotional_reasoning": "Raciocínio Emocional",
```

File: src/locals/pt-br.json [hunk 7]
```diff
@@ -0,0 +1,141 @@
...
+  "emotional_reasoning_explanation_1": "\"Eu me sinto desta forma, portanto sou assim.\" Se estiver justificando o \"perigo\" de algo inofensivo usando o medo como argumento, então provavelmente está usando raciocínio emocional. As coisas não são periogosas porque temos medo delas e também não somos más pessoas só por pensarmos ser.",
+  "emotional_reasoning_explanation_2": "Este é difícil de reconhecer. É necessário algum trabalho para reconhecer que a emoção está dominanando o raciocínio lógico.",
```
Comment (wkurosawa) [inline src/locals/pt-br.json]:
```suggestion
  "emotional_reasoning_explanation_2": "Este é difícil de reconhecer. É necessário algum trabalho para reconhecer que a emoção está dominando o raciocínio lógico.",
```

File: src/locals/pt-br.json [hunk 8]
```diff
@@ -0,0 +1,141 @@
...
+  "emotional_reasoning_thought": "Me Sinto culpado, logo devo ter feito algo de errado.",
+  "fortune_telling": "Predição do Futuro",
+  "fortune_telling_explanation": "Esta distorção ocorre quando prevemos o futuro. Muitas vezes superestimamos as nossas capacidades de prever o que vai acontecer. Frequentemente nos preocupamos com algo que pode acontecer e procuramos evidências que suportem isso.",
+  "fortune_telling_thought": "O avião que estou embarcando vai sofrer um acidente.",
+  "labeling": "Rotulagem",
+  "labeling_explanation": "Se considerarmos uma característica de alguém e reduzirmos a pessoa a essa característica, então estamos rotulando. Se alguém nos desdenhou, não são necessariamente idiotas. Podem estar apenas com pressa. Isto é, também, aplicável a nós mesmos; Cometer um erro não implica que somos uma \"falha\".",
+  "labeling_thought": "Falhei num teste, portanto sou um estudante ruim.",
+  "magnification_of_the_negative": "Maximização do negativo",
```
Comment (wkurosawa) [inline src/locals/pt-br.json]:
```suggestion
  "magnification_of_the_negative": "Maximização do Negativo",
```

File: src/locals/pt-br.json [hunk 9]
```diff
@@ -0,0 +1,141 @@
...
+  "magnification_of_the_negative_explanation": "Se você consideras apenas os aspectos negativos para avaliar uma situação e não considera os aspectos positivos, então  provavelmente está aplicando a distorção de maximização do negativo. Se você está constantemente se julgando por ter falhado numa entrevista de emprego, então está filtrando todas as experiências positivas que você ganhou dessa entrevista.",
```
Comment (wkurosawa) [inline src/locals/pt-br.json]:
```suggestion
  "magnification_of_the_negative_explanation": "Se você consideras apenas os aspectos negativos para avaliar uma situação e não considera os aspectos positivos, então provavelmente está aplicando a distorção de maximização do negativo. Se você está constantemente se julgando por ter falhado numa entrevista de emprego, então está filtrando todas as experiências positivas que você ganhou dessa entrevista.",
```

File: src/locals/pt-br.json [hunk 10]
```diff
@@ -0,0 +1,141 @@
...
+  "magnification_of_the_negative_thought": "Tive uma dieta saudável esta semana, mas não fiz exercício físico.",
+  "mind_reading": "Leitura da mente",
```
Comment (wkurosawa) [inline src/locals/pt-br.json]:
```suggestion
  "mind_reading": "Leitura da Mente",
```

File: src/locals/pt-br.json [hunk 11]
```diff
@@ -0,0 +1,141 @@
...
+  "mind_reading_explanation": "Se estamos preocupados com o que outras pessoas pensam sobre nós, então estamos tentando fazer uma leitura da mente. A menos que alguem nos diga exatamente o que estão pensando, não há forma de saber. Porquê assumir o pior?",
```
Comment (wkurosawa) [inline src/locals/pt-br.json]:
```suggestion
  "mind_reading_explanation": "Se estamos preocupados com o que outras pessoas pensam sobre nós, então estamos tentando fazer uma leitura da mente. A menos que alguém nos diga exatamente o que estão pensando, não há forma de saber. Por que assumir o pior?",
```

File: src/locals/pt-br.json [hunk 12]
```diff
@@ -0,0 +1,141 @@
...
+  "mind_reading_thought": "Acho que fui desagradável com o Jorge, aposto que ele me odeia.",
+  "minimization_of_the_positive": "Minimização do positivo",
```
Comment (wkurosawa) [inline src/locals/pt-br.json]:
```suggestion
  "minimization_of_the_positive": "Minimização do Positivo",
```

File: src/locals/pt-br.json [hunk 13]
```diff
@@ -0,0 +1,141 @@
...
+  "minimization_of_the_positive_explanation": "Se minimizarmos as coisas positivas que nos acontecem, estamos aplicando a distorção da minimização do positivo. Mesmo que o dia não tenha sido 100% da forma que desejavamos, não significa que os 60% que foram bem devam ser ignorados.",
```
Comment (wkurosawa) [inline src/locals/pt-br.json]:
```suggestion
  "minimization_of_the_positive_explanation": "Se minimizarmos as coisas positivas que nos acontecem, estamos aplicando a distorção da minimização do positivo. Mesmo que o dia não tenha sido 100% da forma que desejávamos, não significa que os 60% que foram bem devam ser ignorados.",
```

File: src/locals/pt-br.json [hunk 14]
```diff
@@ -0,0 +1,141 @@
...
+  "other_blaming_explanation_2": "Isto não quer dizer que deveria culpar a si próprio por todas as situações negativas. Você não tem que culpar ninguém. Ninguém tem de ser culpado se não atribuir a culpa às situações e as deixar passar.",
+  "other_blaming_thought": "Aquele idiota está demorarando muito tempo na fila. Assim vou chegar tarde!",
+  "over_generalization": "Generalização",
+  "over_generalization_explanation": "Se tiramos conclusões com apenas um exemplo, estamos generalizando. Se você teve uma falha numa apresentação e assume que isso significa que você é mal em apresentações, então está generalizando.",
+  "over_generalization_thought": "Ninguém me chamou para dançar, o que significa que nunca ninguém me vai chamar.",
+  "self_blaming": "Personalização",
+  "self_blaming_explanation": "Se atribui a totalidade de uma situação negativa como sua culpa, então está aplicando a distorção da personalização. Você não precisa ser responsável por todos os aspectos negativos que acontecem. Se você está preso no trânsito e começa a se culpar por não ter saido de casa mais cedo, está a personaliznado. Trataria outros desta mesma forma?",
```
Comment (wkurosawa) [inline src/locals/pt-br.json]:
```suggestion
  "self_blaming_explanation": "Se atribui a totalidade de uma situação negativa como sua culpa, então está aplicando a distorção da personalização. Você não precisa ser responsável por todos os aspectos negativos que acontecem. Se você está preso no trânsito e começa a se culpar por não ter saído de casa mais cedo, está personalizando. Você trataria outros desta mesma forma?",
```

File: src/locals/pt-br.json [hunk 15]
```diff
@@ -0,0 +1,141 @@
...
+  "self_blaming_thought": "O meu filho indo mal na escola, certamente a culpa é minha.",
```
Comment (wkurosawa) [inline src/locals/pt-br.json]:
```suggestion
  "self_blaming_thought": "O meu filho está indo mal na escola, certamente a culpa é minha.",
```

**Comment by Flaque** [comment]
AMAZING!!! 
PinkDook

**Pull request merged**